release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -166,12 +166,14 @@ For more information, see xref:../storage/container_storage_interface/persistent
 
 The Google Cloud Platform (GCP) persistent disk (PD) CSI driver is automatically deployed and managed on GCP environments, allowing you to dynamically provision these volumes without having to install the driver manually. The GCP PD CSI Driver Operator that manages this driver is in Technology Preview.
 
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc#persistent-storage-csi-gcp-pd[GCP PD CSI Driver Operator].
+
 [id="ocp-4-5-persistent-storage-csi-cinder"]
-==== Persistent storage using the OpenStack Cinder CSI Driver Operator (Technology Preview)
+==== Persistent storage using the OpenStack Cinder CSI Driver Operator
```

Comment:
If this is no longer a TP feature, does it need a little note about that like the one above (line 158)?

Reply:
It's actually a new feature (there was a question as to whether it would land as TP or GA that is now resolved), so not in this case but I appreciate the check. :)